### PR TITLE
Update changelog and pubspec for version bump, flutter format

### PR DESCRIPTION
**Pull request by domesticmouse**
PTAL @KagurazakaHanabi

### CODE REVIEW

File: lib/src/_functions_io.dart [hunk 1]
```diff
@@ -11,7 +11,8 @@ import 'package:flutter/widgets.dart';
 import 'style_sheet.dart';
 import 'widget.dart';
 
-typedef Widget ImageBuilder(Uri uri, String imageDirectory, double width, double height);
+typedef Widget ImageBuilder(
+    Uri uri, String imageDirectory, double width, double height);
```
Comment (KagurazakaHanabi) [inline lib/src/_functions_io.dart]: Please format with `flutter format —line-length 100`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## 0.3.1
+
+ * Expose `tableColumnWidth`
+ * Add `MarkdownStyleSheet.fromCupertinoTheme`
+ * Fix `MarkdownStyleSheet.blockquote`
+ * Flutter for web support
+ * Rebuilt `example/android` and `example/ios` directories
+
```
Comment (KagurazakaHanabi) [inline CHANGELOG.md]: * Add `physic` and `shrinkWrap` to `Markdown` widget
* Add `MarkdownBody.fitContent`
* Support select text to copy
* Fix list bullet alignment

File: lib/src/widget.dart [hunk 3]
```diff
@@ -164,20 +168,20 @@ class _MarkdownWidgetState extends State<MarkdownWidget> implements MarkdownBuil
       encodeHtml: false,
     );
     final MarkdownBuilder builder = MarkdownBuilder(
-      delegate: this,
-      selectable: widget.selectable,
-      styleSheet: styleSheet,
-      imageDirectory: widget.imageDirectory,
-      imageBuilder: widget.imageBuilder,
-      checkboxBuilder: widget.checkboxBuilder,
-      fitContent: widget.fitContent
```
Comment (KagurazakaHanabi) [inline lib/src/widget.dart]: missing tail comma

File: lib/src/_functions_io.dart (hunk 1 above)
Comment (domesticmouse) [reply lib/src/_functions_io.dart]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (domesticmouse) [reply CHANGELOG.md]: Added

File: lib/src/widget.dart (hunk 3 above)
Comment (domesticmouse) [reply lib/src/widget.dart]: Added

**Comment by domesticmouse** [comment]
Thanks for the detailed review. I've just published v0.3.1, it should be up on pub in a couple of hours.

**Pull request merged**